connectivity/FEATURE_BLE/libraries/cordio_stack/ble-host/sources/stack/att/atts_sign.c

```diff
@@ -93,6 +93,54 @@ static attsSignCcb_t *attsSignCcbByConnId(dmConnId_t connId)
   return &attsSignCb.ccb[connId - 1];
 }
 
+/*************************************************************************************************/
+/*!
+ *  \brief  Perform required permission and security checks for a signed write command.
+ *
+ *  \param  connId      Connection ID.
+ *  \param  handle      Attribute handle.
+ *  \param  permissions Attribute permissions.
+ *  \param  pSignCcb    Pointer to ATTS signed PDU connection control block.
+ *
+ *  \return ATT_SUCCESS if successful or error code on failure.
+ */
+/*************************************************************************************************/
+static uint8_t attsSignedWritePermissions(dmConnId_t connId, uint16_t handle, uint8_t permissions,
```

Comment:
I think we were doing these checks in our port. Can you verify if they are still needed in BLE API ?